src/body.js

```diff
@@ -9,10 +9,10 @@ import Stream, {PassThrough} from 'stream';
 import {types, deprecate} from 'util';
 
 import Blob from 'fetch-blob';
+import {FormData, formDataToBlob} from 'formdata-polyfill/esm.min.js';
```

Comment:
Hmmm, will this break `instanceof` if someone else have imported the `FormData` from as so: `import { FormData } from 'formdata-polyfill'`? Maybe that doesn't matter since our public API doesn't specify how `instanceof` will work with `FormData` 🤔 

Yeah, it probably doesn't matter? 🤔

Reply:
My FormData package uses duck type checking: [Symbol.hasInstance](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol/hasInstance)

so as long as it has all the same properties then it can match any kind of instances

Reply:
I suppose we could replace this hole section https://github.com/node-fetch/node-fetch/blob/0f2f9bda685a0e259be50f4ba4aef9f40d38ab2b/src/utils/is.js#L48-L68 code and replace it with instance of FormData check instead...